### DISCUSSION TITLE
Include CNN based NN for TS Forecasting

### DISCUSSION THREAD

**Discussion by DAT-FYAYC** [Nixtla/neuralforecast#405] [Ideas]
Are there any intensions to include CNN based NN as well?
e.g. CNN-QR by Amazone or just simple CNNs
Comparison of CNN-QR: "COMPARISON ANALYSIS OF FACEBOOK’S PROPHET, AMAZON’S DEEPAR+ AND CNN-QR ALGORITHMS FOR SUCCESSFUL REAL-WORLD SALES FORECASTING": https://arxiv.org/abs/2105.00694

This would provide a more complete picture of neural time series forecasting.

**Comment by kdgutier** [comment]
Hey @davidtiefenthaler,

We have TCN based architectures: https://nixtla.github.io/neuralforecast/models.tcn.html
You can combine it with the Multi-Quantile Loss to obtain models like the CNN-QR: https://nixtla.github.io/neuralforecast/losses.pytorch.html#multi-quantile-loss-mqloss

Here is a probabilistic forecasting example:
https://nixtla.github.io/neuralforecast/examples/uncertaintyintervals.html

**Reply by DAT-FYAYC** [reply]
That´s great! Thank you for the hints. This gives a lot of room to use CNN based architectures.
